packages/RichText-Core.package/RTECodeComponent.class/instance/initialize.st

```diff
@@ -2,11 +2,10 @@ initialization
 initialize
 
 	super initialize.
-	self coreObject 
-		addApplicationStyles: (
-			(PHStyle for: #textArea) 
+     " (FontChooserTool new fontList includes: 'SourceCodePro')
```

Comment:
remove code from here because it's not needed anymore

Reply:
isn't there on my version, weird